codegenerator/cli/templates/static/codegen/src/eventFetching/chainWorkers/HyperFuelWorker.res

```diff
@@ -329,12 +329,6 @@ module Make = (
       //set height and next from block
       let currentBlockHeight = pageUnsafe.archiveHeight
 
-      logger->Logging.childTrace({
```

Comment:
Is this just not a useful log? I guess no need to include it since we have metrics.

Reply:
No, I just moved it to the ChainWorker module, which is applied for both HyperFuelWorker/HyperSyncWorker/RPCWorker responses